core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -553,6 +553,9 @@ private void load(String ufsPath, long mountId, long length) {
     String fileId = new AlluxioURI(ufsPath).hash();
     ByteBuf buf = PooledDirectNioByteBuf.allocate((int) (4 * mPageSize));
     try (BlockReader fileReader = createFileReader(fileId, 0, false, options)) {
+      // cache file metadata
+      getFileInfo(ufsPath, GetStatusPOptions.getDefaultInstance());
```

Comment:
I think in our landing case it would be fine to not refresh metadata since there's no metadata in Dora when we start a machine learning training job. What do you think @huanghua78

Reply:
We have in-memory cache and in-RocksDB cache.
If the Alluxio worker is newly started (or re-started), there is no in-memory cache.
But If the worker is re-started from previous run, there might be in-RocksDB cache.
The user should make sure that those targeted files are not changed since last time they were cached.
Currently we don't have a callback to invalidate them. Users can actively invalidate/refresh them here if they want to avoid access stable metadata/data in case that those files really have the possibility of having been modified.

OK. Let me conclude:
1. If the workload is really read-only, and the targeted files/objects are not re-written/updated, it's fine to say that the metadata will not be stale. So, code is fine.
2. If there is some possible way to update the target files/objects before `load()`, and users should do something similar "active metadata sync", to invalidate and refresh the metadata (along with it, the data will be invalidated, too). This can be by user, or in this place by Alluxio loader.
```
     getFileInfo(ufsPath, GetStatusPOptions.newBuilder()
          .setCommonOptions(FileSystemMasterCommonPOptions.newBuilder()
                            .setSyncIntervalMs(0)
                            .build())
          .build());
```

Reply:
updated. Eventually, we need a dedicated load method to determine whether to evict old data & load necessary data based on lastModify time/content hash